silverpeas/content.md

```diff
@@ -0,0 +1,179 @@
+
+-	the open-source, powerful and recommended PostgreSQL database system,
+-	the Microsoft SQLServer database system,
+-	the Oracle database system.
+
+The Silverpeas images support actually only the two first database systems; because of the non-free licensing issues with the Oracle JDBC drivers, Silverpeas cannot include these drivers by default and consequently it cannot use transparently an Oracle database system as a persistence backend.
+
+For the same reasons, the Docker images of Silverpeas aren't shipped with the Oracle JVM but with the default OpenJDK. Silverpeas uses the Wildfly application server as runtime.
+
+The Silverpeas images use the following environment variables to set the database access parameters:
+
+-	`DB_SERVERTYPE` to specify the database system to use with Silverpeas: POSTGRESQL for PostgreSQL, MSSQL for Microsoft SQLServer, ORACLE for Oracle. By default, it is set to POSTGRESQL.
+-	`DB_SERVER` to specify the IP address or the name of the host on which the database system is running. By default, it is set to `localhost`.
+-	`DB_NAME` to specify the database to use with Silverpeas. By default, it is set to `Silverpeas`.
+-	`DB_USER` to specify the user identifier to use by Silverpeas to access the database. By default, it is set to `silverpeas`.
+-	`DB_PASSWORD` to specify the password associated with the user identifier above.
+
+These environment variables can be also defined as properties into the Silverpeas global configuration file `config.properties` (see below).
+
+## Start a Silverpeas instance with a database from a Docker container
+
+In [Docker Hub](https://hub.docker.com/), no Docker images of Microsoft SQLServer are currently available, but you will find a lot of images of PostgreSQL. For example, with an [official PostgreSQL docker image](https://hub.docker.com/_/postgres/), you can start a PostgreSQL instance initialized with a superuser `postgres` with as password `mysecretpassword`:
+
+	$ docker run --name postgresql -d \
+	    -e POSTGRES_PASSWORD="mysecretpassword" \
+	    -v /var/lib/postgresql/data:/var/lib/postgresql/data \
+	    postgres
+
+The directory with the database file is mounted on the host so the data won't be lost when upgrading PostgreSQL to a new version (a Data Volume Container can be used instead).
+
+Once the database system is running, a database for Silverpeas has to be created and a user with administrative rights on this database should be added.
+
+### Start a Silverpeas instance with the default configuration
+
+Finally, a Silverpeas instance can be started by specifying the required database access parameters with the environment variables. In the example, the database is named `Silverpeas` and the priviledged user is `silverpeas` with as password `thesilverpeaspassword`:
+
+	$ docker run --name silverpeas -p 8080:8000 -d \
+	    -e DB_SERVERTYPE="POSTGRESQL" \
+	    -e DB_SERVER="database" \
+	    -e DB_NAME="Silverpeas" \
+	    -e DB_USER="silverpeas" \
```

Comment:
The default user for the `postgres` image is `postgres`, isn't it?  Either this should be `postgres` or the above `docker run` should include `-e POSTGRES_USER=silverpeas`, right? (maybe also `-e POSTGRES_DB=Silverpeas` for good measure?)